chips/nrf52/src/i2c.rs

```diff
@@ -0,0 +1,446 @@
+//! Implementation of I2C for nRF52 using EasyDMA.
+//!
+//! This module supports nRF52's two I2C master (`TWIM`) peripherals,
```

Comment:
This will _never_ be the same controller for the TWIM and TWIS, right? (as opposed to the SAM4L which, for some reason, shares registers between some TWIM/S in a mostly undocumented way)

Reply:
Actually, TWIM, TWIS, SPIM, SPIS share registers and have the same base pointer. It's documented, but not super obvious. I didn't notice until I got half way through this driver.